features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -284,6 +475,119 @@ Scenario: Supplier changes their answers before submission
+  And I do not see the 'Evidence of Provide biscuits' field
+
+  When I choose 'No' radio button for the 'Sip quietly' question
+  And I choose 'Yes' radio button for the 'Provide biscuits' question
+  And I enter 'Only the finest' in the 'Evidence of Provide biscuits' field
+  And I click 'Save and continue'
+  Then I am on the 'Email address the buyer should use to contact you' page
+
+  When I enter 'example-email@gov.uk' in the 'respondToEmailAddress' field
+  And I click 'Save and continue'
+  Then I am on the 'Check your answers' page
+  And I see the 'Your details' summary table filled with:
+      | field               | value                |
+      | Day rate            | £100                 |
+      | Earliest start date | 28/09/17             |
+      | Email address       | example-email@gov.uk |
+  And I see the 'Your essential skills and experience' summary table filled with:
+      | field       | value            |
+      | Boil kettle | Flick the switch |
+      | Taste tea   | Drink the tea    |
+      | Wash mug    | Use soap         |
+      | Dry mug     | fourth evidence  |
+  And I see the 'Your nice-to-have skills and experience' summary table filled with:
+      | field                               | value |
+      | Talk snobbishly about water quality | First nice to have evidence |
+      | Sip quietly                         |                             |
+      | Provide biscuits                    |  Only the finest            |
+  And I see 'Once you submit you can update your application until' text on the page
+  When I click 'Submit application'
+  Then I am on the 'What happens next' page
+  And I see a success banner message containing 'Your application has been submitted.'
+
+
+@skip-staging
+Scenario: Supplier changes their answers after submission
+  Given that supplier has applied to be on that framework
+  And we accept that suppliers application to the framework
+  And that supplier returns a signed framework agreement for the framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
+  And that supplier has filled in their response to that brief but not submitted it
+  And that supplier submits their response to that brief
+
+  When I click the 'View your account' link
+  And I click the 'View your opportunities' link
+  Then I see 'Tea drinker' in the 'Applications you’ve made' summary table
+  And I see the closing date of the brief in the 'Applications you’ve made' summary table
+  And I see 'Submitted' in the 'Applications you’ve made' summary table
+
+  When I click the 'Tea drinker' link
+  Then I am on the 'Check your answers' page
+  When I click a 'Edit' link
+
+  Then I am on the 'What’s the specialist’s day rate?' page
+  And I see '£200' replayed in the question advice
+  And I see '200' as the value of the 'dayRate' field
+
+  When I enter '100' in the 'dayRate' field
```

Comment:
It might be nice to update a few more fields (one of each type maybe, or even maybe all of them) and then check the table for all updated values.

Reply:
The whole point of this scenario was to check that editing a single question in the flow redirects straight back to the Check Your Answers page - if we fill out every question then it would go there by default 😸

Reply:
Ah OK, makes sense.